configuration.md

```diff
@@ -0,0 +1,27 @@
+# Configuration properties
+
+## Main Endpoint
+**/api/config/properties**   
+
+As we don't have yet an use case to iterate over all the configuration properties the main endpoint is not implemented and a 405 error code is returned according to our [general error response codes](README.md#Error codes).
+
+## Single property
+**/api/config/properties/<:property>**
+
+This endpoint provides functionality to retrieve certain (configurable) configuration properties. 
+The DSpace server will have a whitelist of properties that can be retrieved by this endpoint.
+
+
+```json
+{
+  "name": "curate.ui.tasknames",
```

Comment:
please change the example in `google.analytics.key` specifying that the values are always returned as an array also when the property is expected to hold a single value

Reply:
I agree with @abollini that the examples make more sense if we use `google.analytics.key` as the main example.

I think the list of Curation Tasks may not be the best example, since Curation Tasks will be implemented via Scripts & Processes....and I therefore would expect that the list of Curation Tasks would be returned via the Scripts & Processes endpoint (more than likely), and *not* from a separate configuration endpoint.

So, I agree with Andrea that Curation Tasks should likely not be enabled from this `/api/config/properties` endpoint, but that there are some backend properties like Google Analytics keys that would be nice to share to the frontend.